src/components/maintenance/maintenance-documents-tab.js

```diff
@@ -352,15 +306,15 @@ function MaintenanceDocumentsTab({ documents, onDocumentsChange }) {
       key: 'tags',
       render: renderCellTags,
       responsive: ['lg'],
-      width: '300px'
+      width: '200px'
```

Comment:
It creates quite some huge rows when there are many tags (see example from live data). I was wondering if we should not leave the value here and let the browser just take away more space from the title column :thinking:

![Screenshot from 2023-11-27 14-24-12](https://github.com/educandu/educandu/assets/2676617/7723038d-16ee-4d4e-a34e-c32b729126cc)

Reply:
Sure, streamlined "Tags" columns to 300px (they were not the same width before).

Reply:
I read "streamlined", I'm happy.